bionic/deriver.py

```diff
@@ -281,12 +396,27 @@ def _obtain_entity_def_for_dnode(self, dnode):
         """
         Returns an EntityDefinition object for a given descriptor node. If the node is
         an entity, this returns the definition specified by the user; if the node is a
-        tuple, we generate a synthetic definition.
+        draft or tuple, we generate a synthetic definition.
         """
 
         if isinstance(dnode, ast.EntityNode):
             return self._flow_state.get_entity_def(dnode.to_descriptor())
 
+        elif isinstance(dnode, ast.DraftNode):
+            child_entity_def = self._obtain_entity_def_for_dnode(dnode.child)
+            doc_prefix = "(Intermediate value)"
+            if child_entity_def.doc is None:
+                doc = doc_prefix
+            else:
+                doc = doc_prefix + " " + child_entity_def.doc
+            return EntityDefinition(
+                name=dnode.to_descriptor(),
+                protocol=NonSerializableObjectProtocol(),
```

Comment:
nit: how do you feel about using a global instance for this protocol?

Reply:
I don't feel strongly either way. Happy to change it.